src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -898,12 +971,19 @@
   "Given the configuration map from the Puppet Server config
    service return a map with of all the CA settings."
   [{:keys [puppetserver jruby-puppet certificate-authority]}]
-  (-> (select-keys puppetserver (keys CaSettings))
-      (assoc :ruby-load-path (:ruby-load-path jruby-puppet))
-      (assoc :gem-path (str/join (System/getProperty "path.separator")
-                                 (:gem-path jruby-puppet)))
-      (assoc :access-control (select-keys certificate-authority
-                                          [:certificate-status]))))
+    (let [certificate-authority-defaults (set-ca-config-defaults certificate-authority (fs/parent (:cacrl puppetserver)))]
+      (log/debug (str (i18n/trs "config->ca-settings value of certificate-authority-defaults :")
```

Comment:
In general most of these log messages seem better suited for understanding the flow of the code when you're writing a feature, rather than debugging a production issue.